common/scripted_effects/wc_scripted_effects.txt

```diff
@@ -2092,116 +2092,6 @@ create_religious_head_effect = {
 	}
 }
 
-plant_world_tree_effect = {
-	event_target:target_druid = {
-		show_scope_change = no
-		if = {
-			limit = { religion_group = druidism_group }
-			religion_authority = { modifier = new_world_tree }
-		}
-	}
-	event_target:target_title = {
-		show_scope_change = no
-		set_title_flag = world_tree_title_flag
-		if = {
-			limit = { title = b_nordrassil }
-			add_building = tp_nordrassil
-		}
-		else = {
-			add_building = tp_world_tree
-		}
-	}
-	event_target:target_province = {
-		show_scope_change = no
-		if = {
-			limit = { ROOT = { character = event_target:target_druid } }
-			if = {
-				limit = { NOT = { num_of_max_settlements = 6 } }
-				add_holding_slot = 2
-			}
-			else_if = {
-				limit = { NOT = { num_of_max_settlements = 7 } }
-				add_holding_slot = 1
-			}
-		}
-	}
-}
-corrupt_world_tree_effect = {
```

Comment:
You removed this effect, but it's still used in events. Also, you added an effect with the same name, but it does different thing.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/70820629-8fcba300-1df2-11ea-86d5-bbc33141dbe5.png)

</details>